code/modules/nodes/xenocastes/crusher.dm

```diff
@@ -0,0 +1,38 @@
+
+/mob/living/carbon/xenomorph/crusher/ai
+	ai_datum = new/datum/ai_behavior/xeno/crusher
+
+/mob/living/carbon/xenomorph/crusher/ai/Initialize()
+	..()

```

Comment:
Always use `. = ..()` on stuff that's not a datum because else there will be issues with `Initialize()`.